data/bots/lib/libbot/protected-areas.ts

```diff
@@ -4,13 +4,57 @@ import { IUCNCategory } from "./scenarios.ts";
 import { logDebug, logError, logInfo } from "./logger.ts";
 import { tookMs } from "./util/perf.ts";
 
+export enum ProtectedAreaKind {
+  Global = "global",
+  Project = "project",
+}
+
+interface ProtectedAreaSelection {
+  id: IUCNCategory;
```

Comment:
formally it is `id: IUCNCategory | string;`

Reply:
good catch, thank you! fixed in f72eb072fa3fc18e8ed2672f218e97111acb46f3